ROADMAP.md

```diff
@@ -4,55 +4,132 @@ This document provides information on OpenEBS development in current and upcomin
+OpenEBS follows a lean project management approach by splitting the development items into current, near term and future categories. We use GitHub [Projects](https://github.com/orgs/openebs/projects) for tracking the feature development. This document is reviewed and updated based on the [monthly community product roadmap review meetings](https://hackmd.io/S_P-3obgTlO5sBbQYgKbRQ?view). 
 
 ## Current
 
-These are some of the areas under active development planned to be completed within three releases. OpenEBS follows a monthly release cadence with a new release on the 15th of every month. 
-
-While the following are planned items, a higher priority is given to usability and stability issues reported by the community. For the most current issue plan and status check out the [release milestones](https://github.com/openebs/openebs/milestones). 
-
-* [Jiva](https://github.com/orgs/openebs/projects/1)
-  * Optimizations to the replica rebuild process
-  * CSI Driver
-* [cStor](https://github.com/orgs/openebs/projects/9) 
-  * Support for local snapshot and clone via Velero
-  * New cStor API schema with automated Day 2 operations. 
-  * CSI Driver
-  * Support migration from SPC (older schema) to CSPC (new schema)
-* [OpenEBS Local PV - hostpath and device](https://github.com/orgs/openebs/projects/11)
-  * Generating Metrics 
-  * Capacity based scheduling 
-* [OpenEBS ZFS Local PV](https://github.com/orgs/openebs/projects/10)
-  * Generating metrics
-  * Integration and end-to-end tests
-  * Automated install and upgrades  
-* [NDM Enhancements](https://github.com/orgs/openebs/projects/2)
-  * Enhance the discovery probes to support partitions, lvms and so forth
-  * Support Prometheus exporter on block device metrics
-  * Add gRPC API layer around NDM capabilities
-* [Mayastor](https://github.com/openebs/Mayastor)
-  * Replication across multiple nodes
-  * Integration and end-to-end tests
-  * Setup CI
-* ARM based builds for OpenEBS [#1295](https://github.com/openebs/openebs/issues/1295)
+These are some of the backlogs that are prioritized and planned to be completed within the next three releases. While the following are planned items, higher priority is given to usability and stability issues reported by the community. The completion of these items also depends on the availability of contributors.
+
+Note: OpenEBS follows a monthly release cadence with a new release on the 15th of every month.  For the most current plan and status check out the [release project trackers](https://github.com/orgs/openebs/projects) or the component specific trackers listed below. 
+
+### NDM 
+- [Project Tracker](https://github.com/orgs/openebs/projects/2)
+- Source repositories
+  - https://github.com/openebs/node-disk-manager
+- Backlogs
+  - Enhance the discovery probes to support partitions, lvms and so forth
+  - Scan for device media errors, reporting and alerting by enhancing the ndm-exporter
```

Comment:
Are we tracking only backlog items here. Because the exporter and metrics part are currently in the works.

Reply:
This is the same, added the feature that will be enabled to users using the enhancements being worked on for exporter and metrics.